WordPress/src/main/java/org/wordpress/android/ui/sitecreation/NewSiteCreationActivity.kt

```diff
@@ -84,7 +91,7 @@ class NewSiteCreationActivity : AppCompatActivity(),
                         screenTitle,
                         target.wizardState.segmentId!!
                 )
-            DOMAINS -> NewSiteCreationDomainFragment.newInstance(screenTitle, "Test site")
+            DOMAINS -> NewSiteCreationDomainsFragment.newInstance(screenTitle)
```

Comment:
Oh interesting. I did not know that. I can easily implement this one if we want to @SylvesterWilmott.

Reply:
Ah yes, we wanna keep that for sure 👍

Reply:
I've made this change in 960cd6b7dacc44f1397eed6aeb64b51bb5b20f84. I have a couple questions about this:

* Should we initially set the edit text's query to the site title? This was not the case in the current domains page, so I didn't implement it.
* When there is a title for the site, the header will not be visible initially as shown in this gif: https://cloudup.com/ckxlk7phfNW. Should we add a delay to the initial request so it'll be visible? On the other hand, the problem with a delay is that there is no user action, so it might feel kind of weird.

I like the idea behind this feature, but I am not sure if it feels right when I am using it. What do you think @SylvesterWilmott @malinajirka ?

Reply:
Thanks Oguz for the changes;)

It's Sylvester's decision, however, I'll just share my opinion.

>Should we initially set the edit text's query to the site title? This was not the case in the current domains page, so I didn't implement it.

I think we should leave it empty.

>When there is a title for the site, the header will not be visible initially as shown in this gif: https://cloudup.com/ckxlk7phfNW. Should we add a delay to the initial request so it'll be visible? On the other hand, the problem with a delay is that there is no user action, so it might feel kind of weird.

I think one of the reasons why we hide the header is due to the fact that the keyboard is taking up a lot of space. However, when we enter the screen the keyboard is not shown, so I feel like we can show both the suggestions and the header. Wdyt?

Reply:
> I think one of the reasons why we hide the header is due to the fact that the keyboard is taking up a lot of space. However, when we enter the screen the keyboard is not shown, so I feel like we can show both the suggestions and the header. Wdyt?

That could work. Let's wait for @SylvesterWilmott before we do anything about this one though.

Reply:
>Should we initially set the edit text's query to the site title? This was not the case in the current domains page, so I didn't implement it

It should be left empty

>When there is a title for the site, the header will not be visible initially as shown in this gif: https://cloudup.com/ckxlk7phfNW. Should we add a delay to the initial request so it'll be visible? On the other hand, the problem with a delay is that there is no user action, so it might feel kind of weird.

We can show the header when the view is loaded right? Focusing the field causes it to hide and the field should not be focused right away since we have the suggestions that we don't want to hide with a keyboard initially. I think in the V2 we should trigger the header hiding by typing a character, then we're not limiting ourselves to not being able to auto-focus things.

Reply:
OK, I'll try to bind this to the query shown in the UI in a separate PR and attach a gif for it. Thank you both for your feedback!